commons/src/main/java/org/sonarsource/analyzer/commons/RuleMetadataLoader.java

```diff
@@ -206,15 +207,31 @@ Map<String, Object> getMetadata(String ruleKey) {
     }
   }
 
-  private static void setCodeAttributeFromJson(NewRule rule, Map<String, Object> code) {
+  private void setCodeAttributeFromJson(NewRule rule, Map<String, Object> code) {
     String attribute = getString(code, "attribute");
     rule.setCleanCodeAttribute(CleanCodeAttribute.valueOf(attribute));
 
     Map<String, String> impacts = (Map<String, String>) code.get("impacts");
     if (impacts == null || impacts.isEmpty()) {
       throw new IllegalStateException(String.format(INVALID_PROPERTY_MESSAGE, "impacts") + " for rule: " + rule.key());
     }
-    impacts.forEach((softwareQuality, severity) -> rule.addDefaultImpact(SoftwareQuality.valueOf(softwareQuality), Severity.valueOf(severity)));
+    impacts.forEach((softwareQuality, severity) -> rule.addDefaultImpact(SoftwareQuality.valueOf(softwareQuality),
+      getCleanCodeTaxanomySeverity(severity)));
```

Comment:
In terms of style, I would put either both parameters of `addDefaultImpact()` in distinct lines or everything on the same line. This mixed mode is not the best in terms of readability.